src/entity/src/user.js

```diff
@@ -785,18 +807,40 @@ export class User {
    * @return {Promise<Object>} The response.
    */
   async forgotUsername(options = {}) {
```

Comment:
@ThomasConner  In the interest of avoiding confusion, should we deprecate this instance method? Seems like it's meaningless anyway to call `forgotUsername` when you already have a user instance. I have similar comments about a couple of other user verification APIs. We could start by adding deprecation notes and eventually taking them out. Thoughts?